frontend/views/pages/GroupChat.vue

```diff
@@ -82,16 +82,23 @@ page(pageTestName='groupChat' :miniHeader='isDirectMessage()')
           @click='editDescription'
         ) Add description
 
-  template(#sidebar='')
+  template(#sidebar='{ toggle }')
     chat-nav(:title='L("Chat")')
       conversations-list(
         :title='L("Channels")'
         routepath='/group-chat/'
         :list='channels'
         route-name='GroupChatConversation'
+        @new='toggle'
+        @redirect='toggle'
       )
 
-      chat-members(:title='L("Direct Messages")' action='addDirectMessage')
+      chat-members(
+        action='addDirectMessage'
+        :title='L("Direct Messages")'
+        @new='toggle'
+        @redirect='toggle'
+      )
```

Comment:
What is this `toggle`? Where is it defined?

Reply:
Here, in `views/components/Page.vue`.
```
-     slot(name='sidebar')
+     slot(name='sidebar' :toggle='toggleMenuIfTouch')
```